src/command/Parser.java

```diff
@@ -185,17 +188,43 @@ private static void ProcessTodo(String input, TaskList tasklist, Ui ui){
      * @param tasklist Tasklist of the user.
      * @param ui Ui that interacts with the user.
      */
-    private static void ProcessEvent(String input, TaskList tasklist, Ui ui){
+    private static void ProcessDoAfter(String input, TaskList tasklist, Ui ui){
```

Comment:
Please take care of the documentation as well while making changes to the code! For now the documentation seems messy. For example above the new ProcessDoAfter function, the documentation actually belongs to function ProcessEvent. This might cause confusion. Overall the changes seems good!